integration_tests/CMakeLists.txt

```diff
@@ -59,8 +59,8 @@ macro(RUN)
         if (KIND STREQUAL "llvm")
             add_custom_command(
                 OUTPUT ${name}.o
-                COMMAND lpython -c ${name}.py -o ${name}.o
-                DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/${name}.py
+                COMMAND lpython -c ${CMAKE_SOURCE_DIR}/${name}.py -o ${name}.o
```

Comment:
Isn't `CMAKE_CURRENT_SOURCE_DIR` equivalent to `CMAKE_SOURCE_DIR` in this case, because we do not use `add_subdirectory`?

Reply:
Yeah, they are same.